src/Lykke.Service.PayInvoicePortal.Repositories/AutofacModule.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Autofac;
+using AzureStorage.Tables;
+using Common.Log;
+using Lykke.Service.PayInvoicePortal.Core.Repositories;
+using Lykke.SettingsReader;
+
+namespace Lykke.Service.PayInvoicePortal.Repositories
+{
+    public class AutofacModule : Module
+    {
+        private readonly IReloadingManager<string> _subscriptionConnectionString;
+        private readonly ILog _log;
+
+        public AutofacModule(IReloadingManager<string> subscriptionConnectionString, ILog log)
+        {
+            _subscriptionConnectionString = subscriptionConnectionString;
+            _log = log;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance<ISubscriberRepository>(
```

Comment:
Register SubscriberRepository as builder.RegisterType< SubscriberRepository  >.As< ISubscriberRepository > and 
AzureTableStorage as a RegisterInstance. It allow you write unit tests and substitute in-memory storage.

Reply:
Yes, sure. Thanks for your advice, but in this case I guess it's Okay.